internal/config/settings.go

```diff
@@ -155,6 +155,62 @@ func init() {
 	VolatileSettings = make(map[string]bool)
 }
 
+func validateParsedSettings() error {
+	var err error
+	defaults := DefaultAllSettings()
+	for k, v := range parsedSettings {
+		if strings.HasPrefix(reflect.TypeOf(v).String(), "map") {
+			if strings.HasPrefix(k, "ft:") {
+				for k1, v1 := range v.(map[string]interface{}) {
+					if _, ok := defaults[k1]; ok {
+						if e := verifySetting(k1, v1, defaults[k1]); e != nil {
+							err = e
+							parsedSettings[k].(map[string]interface{})[k1] = defaults[k1]
+							continue
+						}
+					}
+				}
+			} else {
+				if _, e := glob.Compile(k); e != nil {
+					err = errors.New("Error with glob setting " + k + ": " + e.Error())
```

Comment:
In this case, unlike in other cases, we leave the invalid glob in place, whereas later in `InitLocalSettings()` we assume that the glob is valid and don't check `glob.Compile()` for errors?